kubernetes/leapp-scratch-image/6/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM scratch
+
+ADD leapp6.tar /
+
+CMD ["/sbin/init"]
```

Comment:
The reason we have two images is simply because Centos7 use symlinks in /:
bin -> /usr/bin, 
lib -> /usr/lib, 
lib64 ->/usr/lib

and Centos6 keeps these folders separated. In order to prevent future issues with possible name collisions, we should use separate images for centos6 (upstart based) and centos7 (systend based)